app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -772,7 +768,7 @@ def parse_image_name(image, image_ref)
         {
           :name          => image_parts[:name],
           :tag           => image_parts[:tag],
-          :digest        => image_parts[:digest] || (image_ref_parts[:digest] if image_ref_parts),
+          :digest        => image_parts[:digest] || (image_ref_parts[:digest] if image_ref_parts) || "",
```

Comment:
I didn't know that. Also at the beginning I thought of saving the images in `@data_index[:container_images][:by_repository][...][:by_name][..][:by_digest][..] so it made sense. I left this to allow easier iteration over the images in `@data_index[:container_images]` . I will revert this change now as it seems confusing.